lib/resources/filesystem.rb

```diff
@@ -1,6 +1,7 @@
 module Inspec::Resources
   class FileSystemResource < Inspec.resource(1)
     name 'filesystem'
+    supports platform: 'unix'
```

Comment:
same here change this to linux and remove the os_family one

Reply:
got it